app/logical/source/url/lofter.rb

```diff
@@ -48,6 +48,10 @@ def parse
       @username = $1
       @work_id = work_id
 
+    # https://gengar563.lofter.com/tag/%E5%BA%9F%E5%BC%83%E7%9B%90%E9%85%B8%E5%A4%84%E7%90%86%E5%8E%82
+    in /^([a-z0-9-]+)\.lofter\.com$/, "tag", tag
+      @unescaped_tag = CGI.unescape(tag)
```

Comment:
Unescaping the path should probably happen globally in Danbooru::URL#initialize, but it would need testing to make sure it doesn't break other things.

https://github.com/danbooru/danbooru/blob/master/app/logical/danbooru/url.rb#L42